src/app/view/paperView.js

```diff
@@ -561,6 +563,34 @@ export default class PaperView {
         textLayer.bringToFront();
     }
 
+    /**
+     * Show all but the desired features
+     * Chosen features behind mask
+     * Built for use in uF Guide Tool
+     * @param {Array<paperObject>} features Array of features to be displayed
```

Comment:
So is this paperobjects/features ?

Reply:
@MurrayLF I'm asking because the text says Array of features, I think this will be rectified once you merge the paperview.ts into the codebase

Reply:
The paperObjects are features in the paperView.paperFeatures sense.

Reply:
@MurrayLF can you do the merge then, I think that should solve this issue and the resolve conflict I'm seeing here

Reply:
Merged the current CIDARLAB webpack-build into mine